pam/go-exec/module.c

```diff
@@ -414,44 +425,51 @@ wait_child_thread (gpointer data)
   WaitChildThreadData *thread_data = data;
   pid_t child_pid = thread_data->child_pid;
   int exit_status = PAM_SYSTEM_ERR;
+  GDBusConnection *connection;
 
   while (TRUE)
     {
       int status;
       pid_t ret = waitpid (child_pid, &status, 0);
+      int errsv = errno;
```

Comment:
nit: I would prefer a more descriptive variable name

```suggestion
      int waitpidErrno = errno;
```

Reply:
Well `errsv` it's the [GLib standard](https://gitlab.gnome.org/search?search=errsv&nav_source=navbar&project_id=658&group_id=8&search_code=true&repository_ref=main) (and in general GNU standard) for `err_save` or similar, so I'd keep using it since it should be clear (since we're in a GLib/Gio world here :partying_face:). But let me know if it's a problem for you.

Reply:
Ah I didn't know that. That's fine then of course :)